ztools/x509/extensions_test.go

```diff
@@ -48,7 +51,7 @@ func (s *ExtensionsSuite) SetUpTest(c *C) {
 
 func (s *ExtensionsSuite) TestEncodeDecodeSAN(c *C) {
 	for _, cert := range s.parsedCerts {
-		if (cert.Issuer.CommonName != "SAN Test"){
+		if cert.Issuer.CommonName != "SAN Test" {
```

Comment:
Are you sure this won't cause false positives, e.g. tests passing that shouldn't?